Looped/Interface/User/UserViewController.swift

```diff
@@ -114,3 +128,9 @@ final class UserViewController: ViewController {
         userView.segmentedControl.setTitle("Favorites", forSegmentAt: 1)
     }
 }
+
+extension UserViewController: LoginViewControllerDelegate {
+    func onSuccess() {
+        // Probably useless delegate pattern as observing currentSession works
+    }
```

Comment:
Właściwie to dismiss powinien być tutaj, skoro flow controller działa o odpowiada za flow to niech to tutaj zrobi.